dags/common/cleanup.py

```diff
@@ -1,9 +1,13 @@
+import re
+
 from bleach import clean
 from bleach.html5lib_shim import Filter
 from bleach.sanitizer import Cleaner
 
 
 def clean_whitespace_characters(input):
+    if type(input) == list:
+        return input
     return " ".join(input.split())
```

Comment:
Won't `re.sub` or `str.replace` work for this case? IMO It would be more readable.

Reply:
in my opinion, regex might be a bit difficult, because we need to catch:
 * new lines
 * tabs
 * double spaces
 * trailing spaces
I can find different solution but might become hard to read